_maps/shuttles/infiltrator_basic.dmm

```diff
@@ -206,7 +206,6 @@
 /area/shuttle/syndicate/airlock)
 "aY" = (
 /obj/machinery/door/poddoor{
-	id = "smindicate";
```

Comment:
Probably best to revert this and the button change - the default ID is "1", which is accessible to anyone that decides to make their own pod doors and buttons therein in-game.
I really don't like the idea of the infiltrator's door just being toggleable by any idiot across the entire server.